frontend/src/pages/GranteeRecord/pages/TTAHistory.js

```diff
@@ -1,34 +1,88 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Helmet } from 'react-helmet';
 import Overview from '../../../widgets/DashboardOverview';
+import FilterMenu from '../../../components/FilterMenu';
+
+function expandFilters(filters) {
+  const arr = [];
+
+  filters.forEach((filter) => {
+    const { topic, query, condition } = filter;
```

Comment:
Is this needed for filters like the `role` filter that has a multi-select?

Reply:
I think Express in general expects &role=foo&role=bar rather than &role=foo,bar - seems like it makes more sense to pass it in like that rather than split at the backend. (I could be wrong about either of those things)